var/spack/repos/builtin/packages/py-quantiphy/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyQuantiphy(PythonPackage):
+    """physical quantities (numbers with units)"""
+
+    homepage = "https://quantiphy.readthedocs.io"
+    pypi = "quantiphy/quantiphy-2.20.tar.gz"
+
+    maintainers("ax3l")
+
+    license("MIT", checked_by="ax3l")
+
+    version("2.20", sha256="ba5375ac55c3b90077a793588dd5a88aaf81b2c3b0fc9c9359513ac39f6ed84d")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-flit-core@2:,:3", type=("build"))
```

Comment:
I believe you want the version range `@2:3` here to reflect `pyproject.toml`'s `requires = ["flit_core >=2,<4"]`.

Reply:
Since there don't appear to be any versions within that range that are problematic (https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges).

Reply:
Thanks @tldahlgren !
```suggestion
    depends_on("py-flit-core@2:3", type=("build"))
```